shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -877,13 +912,46 @@ SemanticsObject hitTest(float[] point) {
         boolean isFocusable() {
             int scrollableActions = Action.SCROLL_RIGHT.value | Action.SCROLL_LEFT.value
                     | Action.SCROLL_UP.value | Action.SCROLL_DOWN.value;
+            if (hasFlag(Flag.SCOPES_ROUTE)) {
```

Comment:
Maybe add a comment here that we enforce in the framework that no other useful semantics are added to this node?

Also, maybe makes this the first line of the method? The variable scrollableActions above this kinda belongs to the statement below this.